assembl/static2/js/app/components/debate/thread/post.jsx

```diff
@@ -167,6 +171,29 @@ class Post extends React.PureComponent {
     };
     return (
       <div className="posts" id={id}>
+        {extracts.length
+          ? <div className="extracts" style={{ position: 'absolute', left: '100px', width: '250px', lineHeight: '30px' }}>
+            <div className="badges">
+              {extracts.map((extract) => {
+                return (
+                  <div className="nugget">
+                    <div className="nugget-img">
+                      <span className="assembl-icon-pepite color2" style={{ fontSize: '50px' }}>
+                          &nbsp;
+                      </span>
+                    </div>
+                    <div className="nugget-txt" style={{ padding: '25px 0', marginRight: '20px', fontStyle: 'italic' }}>
+                      <span key={extract.id} style={{ color: '#5404D7' }}>
```

Comment:
Also for the styles here. Also, this colour is defined as `$purple` in `_variables.scss`. Use the variables as much as possible. This will ensure that updating themes in 1 place will be propagated across the entire code-base.

Reply:
same